odc/stats/plugins/lc_fc_wo_a0.py

```diff
@@ -262,8 +236,30 @@ def reduce(self, xx: xr.Dataset) -> xr.Dataset:
 
         xx = xx.groupby("time.month").map(median_ds, dim="spec")
 
-        data = self._veg_or_not(xx)
-        max_count_veg = self._max_consecutive_months(data, NODATA)
+        # consecutive observation of veg
+        veg_data = self._veg_or_not(xx)
+        max_count_veg = self._max_consecutive_months(veg_data, NODATA)
+
+        # consecutive observation of non-veg
+        non_veg_data = expr_eval(
+            "where(a<nodata, 1-a, nodata)",
```

Comment:
I don't understand why it's 1-a

Reply:
1: veg 0: non-veg. after flipping 1: non-veg, 0: veg, easier to use same function to calculate monthly median.